sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -401,6 +381,53 @@ final class DataFrameWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * :: Experimental ::
+   * Starts the execution of the streaming query, which will continually send results to the given
+   * [[ForeachWriter]] as new data arrives. The returned [[ContinuousQuery]] object can be used to
```

Comment:
...as new data arrives. The [[ForeachWriter]] can be used to send the data generated by the DataFrame/Dataset to an external system. The returned...